### DISCUSSION TITLE
Teachers wanted

### DISCUSSION THREAD

**Discussion by Blakezulla** [community/community#110198] [Programming Help] [answered]
I'd like to learn about anything about coding and I'm hoping some of you want to help! Thx

**Comment by ghost** [comment, marked as answer]
You already know javascript I guess, so try to diving deep into it and once you had completely covered the required knowledge then Transition to another language like Python or any other frameworks like React, Vue or Angular if you are interested in Frontend or node.js if interested in backend... If you want to learn any other programming language you can even learn rust, it will surely help you in lot of cases...

**Reply by Blakezulla** [reply]
I've already filled out the FAFSA with plans for classes come fall 😂 Thank you!

**Reply by ghost** [reply]
All the best @Blakezulla
